### ISSUE TITLE
Juche

### ISSUE THREAD

**Issue by ProletarianBanner** [LeftValues/leftvalues.github.io#82]
I had previously filled out a Google Form asking you to add Juche, have you added it? If not, will you?

**Comment by X-Clyder** [comment]
I don't think it needs to be added. There is no need to overload the test with too many ideologies, especially not the most popular ones.
